### DISCUSSION TITLE
最新模板没有交大图标

### DISCUSSION THREAD

**Discussion by ghost** [sjtug/SJTUThesis#795] [提问]
v1.1.0版本编译后似乎封面上没有交大校徽和校名，页眉上也没有交大校徽

**Comment by skyzh** [comment]
只有本科生模版有校徽。

**Comment by dongqing950315** [comment]
在 [SJTUThesis-master\texmf\tex\latex\sjtuthesis\sjtuthesis-graduate.ltx](url)文件中，新增一行中文扉页插入交大logo的命令：

line118: `\begin{center}` 和 line119: `\vspace*{28pt}`之间插入：

`\includegraphics[width=3.13cm] {sjtu-vi-badge-red.pdf}\\`
